generators/base-core/generator.ts

```diff
@@ -1026,18 +1028,63 @@ templates: ${JSON.stringify(existingTemplates, null, 2)}`;
   /**
    * Edit a property file, adding or updating a key.
    */
-  editPropertyFile(file: string, properties: PropertyFileKeyUpdate[], options: EditFileOptions & { sortFile?: boolean } = {}): void {
-    const { sortFile = true, ...editOptions } = options;
+  editPropertiesFile(
```

Comment:
I would prefer it not to be in a generator.ts.

To me generator.ts is made to orchestrate at worse to facade a utils/support/phase package.
They should be empty of business logic and just act as a kind of router or orchestrator.

Doing so would permit generators that do not extend the one facading to consume the utils anyway

Reply:
I think it can be done.
But lets do in another PR.